includes/class-support-overrides.php

```diff
@@ -49,6 +49,11 @@ public function __construct() {
 		if ( class_exists( 'WooCommerce' ) ) {
 			add_filter( 'woocommerce_background_image_regeneration', '__return_false' );
 		}
+		function remove_site_health_test( $tests ) {
+			unset( $tests['direct']['persistent_object_cache'], $tests['direct']['yoast-page-comments-check'] );
+			return $tests;
+		}
+		add_filter( 'site_status_tests', 'remove_site_health_test', 99 );
 	}
```

Comment:
Can you please move this function to inside of the class as a static public function and then call the hook in this init function? I'm making the suggestion in github so linting and whitespace may not be perfect. Please don't use the commit suggestion feature and instead manually commit this after testing on your local. 
```suggestion
		add_filter( 'site_status_tests', array( $this, 'remove_site_health_test', 99 ) );
	}
			public static function remove_site_health_test( $tests ) {
			unset( $tests['direct']['persistent_object_cache'], $tests['direct']['yoast-page-comments-check'] );
			return $tests;
		}
```